amazoncaptcha/devtools.py

```diff
@@ -58,7 +58,7 @@ def _extract_captcha_link(self, captcha_page):
 
         """
 
-        return captcha_page.text.split('<img src="')[1].split('">')[0]
+        return captcha_page.text.split('<img src="')[2].split('" alt')[0]

```

Comment:
This fixes finding the URI, but still feels pretty hacky. Since this is dev-only, bringing in a propery HTML parser like `beautifulsoup4` might be worth it to stabilize this more.